Peek-swiftUI/Modules/MainView/Views/CategoryListView.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  CategoryListView.swift
+//  Peek-swiftUI
+//
+//  Created by mazen eldeeb on 25/08/2024.
+//
+
+import SwiftUI
+
+struct CategoryListView: View {
+    let category: MoviesResponse
+    var body: some View {
+        VStack(alignment: .leading) {
+            HStack {
+                Text(category.categoryTitle)
+                    .bold()
+                    .font(.title)
+                    .foregroundStyle(.white)
+                Spacer()
+            }
+            
+            ScrollView(.horizontal, showsIndicators: false) {
+                LazyHStack {
+                    ForEach(category.results, id: \.id) { movie in
+                        MovieCard(movie: movie)
+                        
+                    }
+                    .listStyle(.plain)
+                }
+            }.frame(width: .infinity, height: Constants.movieCardheight, alignment: .leading)
```

Comment:
this causes runtime warning, dont set width to infinity, infinity can be user with maxWidth for example

Reply:
done